code/game/objects/structures/low_wall.dm

```diff
@@ -415,6 +408,11 @@
 					user.visible_message(SPAN_WARNING("The wall was torn open by [user]!"))
 					return
 
+	//Turn on harm intent to override this behaviour and instead attack the wall
+	if (!(locate(/obj/structure/window) in loc) && user.a_intent != I_HURT)
```

Comment:
Why move it here? Wouldn't it make it impossible to place welders on the low walls, because deconstruction bloc will run first.

Reply:
It fires only if the welder is turned on, because it doesn't have a welding quality otherwise. Which is far less cancerous than harm-only deconstruction.